chapters/license-matching-guidelines-and-templates.md

```diff
@@ -146,7 +187,9 @@ To avoid a license mismatch merely because extraneous text that appears at the e
 
 ### B.13.2 Guideline <a name="B.13.2"></a>
 
-Ignore any text that occurs after the obvious end of the license and does not include substantive text of the license, for example: text that occurs after a statement such as, "END OF TERMS AND CONDITIONS," or an exhibit or appendix that includes an example or instructions on to how to apply the license to your code. Do not apply this guideline or ignore text that is comprised of additional license terms (e.g., permitted additional terms under GPL-3.0, section 7). Templates do not include markup for this guideline.
+Ignore any text that occurs after the obvious end of the license and does not include substantive text of the license, for example: text that occurs after a statement such as, "END OF TERMS AND CONDITIONS," or an exhibit or appendix that includes an example or instructions on to how to apply the license to your code. Do not apply this guideline or ignore text that is comprised of additional license terms (e.g., permitted additional terms under GPL-3.0, section 7). 
+
+To implement this guideline, use the <optional> XML element tag as described in section B.3.5.
```

Comment:
Probably need to put `<optional>` in backticks as with the other tag names above, otherwise I don't think it'll display in the rendered Markdown.

Reply:
yes, got it